java/connecting-to-bigtable/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
```

Comment:
Please use https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/blob/master/java/quickstart/pom.xml#L25 as the parent.
It will ensure all the right linting checks are done.

Reply:
Done